.ci_support/linux_64_.yaml

```diff
@@ -1,8 +1,10 @@
 cdt_name:
-- cos6
+- conda
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
 docker_image:
-- quay.io/condaforge/linux-anvil-cos7-x86_64
+- quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
+- '3.9'
```

Comment:
I understand that 3.9 is the earliest supported version of Python. However, it seems that the minimum python version is earlier than that. For example, see https://github.com/metoppv/improver/blob/master/setup.cfg#L16.

Reply:
Indeed, to be on the safe side, perhaps it would be worth putting up a PR against metoppv/improver that proposes to increase the python lower limit??
(i.e. removal of environment_a and environment_b etc.)
At minimum it would give visibility of this to those it concerns.

Reply:
This `.ci_support/linux_64_.yaml` file is provided by conda-forge, so we don't get to choose the contents of it, such as to add older python versions. It looks like conda-forge are following python [security support lifetimes](https://endoflife.date/python), so 3.8 is now considered too old.

@cpelley Yes, it'd be reasonable to create an issue/PR on metoppv/improver to increase the lower bound version there (eg. in READMEs) and make corresponding changes in the test environments.

Reply:
OK, that makes sense. I've created https://github.com/metoppv/improver/issues/2068 to track this. In the meantime I'm happy to approve this PR.